db/migrate/20161002153425_add_join_table_between_users_and_diary_entries.rb

```diff
@@ -0,0 +1,11 @@
+class AddJoinTableBetweenUsersAndDiaryEntries < ActiveRecord::Migration
+  def change
+    create_table :diary_entry_subscriptions, :id => false do |t|
+      t.column :user_id, :bigint, :null => false
+      t.column :diary_entry_id, :bigint, :null => false
+    end
+
+    add_index :diary_entry_subscriptions, [:user_id, :diary_entry_id], :unique => true, :name => "index_diary_subscriptions_on_user_id_and_diary_entry_id"
```

Comment:
Can we make this `add_primary_key` please and drop the unique and name options. Also we should add the foreign keys - I think this is right:

```
add_foreign_key :diary_entry_subscriptions, :diary_entries, :name => "diary_entry_subscriptions_diary_entry_id_fkey"
add_foreign_key :diary_entry_subscriptions, :users, :name => "diary_entry_subscriptions_user_id_fkey"
```